acceptancetests/assess_cross_model_relations.py

```diff
@@ -70,13 +75,9 @@ def assess_cross_model_relations_multiple_controllers(args):
     first_bs_manager = BootstrapManager.from_args(args)
     with first_bs_manager.booted_context(args.upload_tools):
         first_model = first_bs_manager.client
-        # Need to share juju_data/home
-        second_bs_manager.client.env.juju_home = first_model.env.juju_home
-        with second_bs_manager.existing_booted_context(
-                second_args.upload_tools):
+        with second_bs_manager.booted_context(second_args.upload_tools):
             second_model = second_bs_manager.client
-            ensure_cmr_offer_consumption_and_relation(
-                first_model, second_model)
+            ensure_user_can_consume_offer(first_model, second_model)
```

Comment:
Same comment; is there something more descriptive than first_model and second_model?